src/Raven.Client/Documents/Operations/TimeSeries/TimeSeriesConfiguration.cs

```diff
@@ -9,13 +12,17 @@ public class TimeSeriesConfiguration : IDynamicJson
         internal const char TimeSeriesRollupSeparator = '@';
         public Dictionary<string, TimeSeriesCollectionConfiguration> Collections { get; set; }
 
-        public TimeSpan PolicyCheckFrequency { get; set; } = TimeSpan.FromMinutes(10);
+        public TimeSpan? PolicyCheckFrequency { get; set; }
 
-        internal void Initialize()
+        public TimeSeriesValueNameMapper ValueNameMapper { get; set; }
```

Comment:
I don't like this name. Can we find something more descriptive?

Reply:
suggestions are welcome :)